deployment/ansible/group_vars/vagrant.example

```diff
@@ -2,6 +2,20 @@
 ckan_disqus_shortname: ENTER_SHORTNAME_HERE
 ckan_site_url: http://localhost:8080/
 ckan_production: false
+ckan_package_dir: '/tmp'
+ckan_version: 2.2.3
```

Comment:
The provisioning depends to a fair extent on what version is getting installed--e.g. plugins appear and disappear between versions--so I feel like the version should be in a tracked file.  Right now it kind of is, since the actual package that gets installed is set in `ckan.app/defaults/main.yml`, but obviously this is used by `ckan.database`, so it can't be in a particular role.  I think it should go in `group_vars/all`.  And to avoid mis-match, I think the `ckan.app` role should use it to pick what package to install.

Actually, another complication here is that there aren't separate packages for different patch versions, so it just installs the latest patch version of the specified minor version.  So ideally we'd have a variable to set the minor version (e.g. `ckan_version: 2.2`) and derive the actual patch version to use for checking out the repo in the database role.  But I'm not sure how to do that.  There's a `version-2.2` tag on the repo, but it means 2.2.0, not "latest 2.2".  Probably in practice it wouldn't make a difference if the checked out version used to set the datastore permissions were behind by a patch version or two.

Reply:
Instead of adding logic to look up the patch release (which would've included some hacky, non-idempotent steps to the provisioner), I just added `ckan_patch_version` as its own variable, much like we do with `java_version` and `java_minor_version` in https://github.com/azavea/ansible-java. I also moved `ckan_version` to `group_vars/all` as discussed above.